.github/workflows/snippet_test.yml

```diff
@@ -48,4 +48,4 @@ jobs:
         run: poetry install
 
       - name: Run Snippets
-        run: (for i in snippets/*.py; do echo "Running $i" && poetry run python $i || exit 1; done)
+        run: (for i in snippets/*.py; do echo "Running $i" && poetry run python $i && sleep 5 || exit 1; done)
```

Comment:
Can we add the `sleep` later on? I'd loathe to wait anymore for the completion of CI tests. Hopefully, the changes in the below files should suffice in pacifying the faucet rate limits.

Reply:
A couple minutes is pretty fast, and you usually don't need to worry much about the snippet tests anyways.